tests/test_fixtures/rule_based_profiler/example_notebooks/great_expectations/great_expectations.yml

```diff
@@ -15,31 +15,31 @@ config_version: 3.0
 # add a new datasource. Read more at https://docs.greatexpectations.io/en/latest/reference/core_concepts/datasource.html
 datasources:
   taxi_multi_batch_datasource:
+    execution_engine:
```

Comment:
This config seems to have been rearranged by the new configuration saving functionality.